test-case/run-all-tests.sh

```diff
@@ -18,6 +18,7 @@ speaker
 pause-resume-playback
 pause-resume-capture
 volume
+volume_levels
```

Comment:
Comment it out at the end considering it requires octable/matlab

```shell
# Requires octave
# testlist="$testlist volume_levels"
```

Reply:
Yep, this is a reference implementation for python version of the same. I think I'll add also ramp length (as milliseconds) check, then the Octave part should be ready.

Reply:
Done.